src/typography/typography.css

```diff
@@ -64,6 +68,39 @@ body {
   color: var(--omrs-color-brand-black);
 }
 
+.omrs-type-body-large {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.188rem;
+  line-height: 129%;
+  color: var(--omrs-color-brand-black);
+}
+
+.omrs-type-body-regular {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.0625rem;
+  line-height: 147.6%;
+  color: var(--omrs-color-brand-black);
+}
+
+.omrs-type-body-serif {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.0625;
+  line-height: 147.6%;
+  color: var(--omrs-color-brand-black);
+}
+
+.omrs-type-body-small {
```

Comment:
Looks like we're missing a few classes?

`omrs-type-body-mini`
`omrs-type-body-subheading`

And that maybe `omrs-type-body-subheading` should have `text-transform: uppercase;`?? Based on the designs in figma. Not sure if we should force uppercase or not, but if we do want to do that it is done with text-transform. See https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

Reply:
These two classes were removed by Greg (they have a line through them in the figma file)

Reply:
ah okay 👍